requirements.txt

```diff
@@ -2,4 +2,5 @@ tqdm>=4.62.3,<5
 fsspec>=2022.1.0
 numpy>=1.19.5,<2
 pandas>=1.1.5,<2
-pyarrow>=6.0.1,<8
\ No newline at end of file
+pyarrow>=6.0.1,<8; python_version < "3.11"
```

Comment:
We could bump that, but I prefer to leave it unchanged for the users of the lib in case it breaks their dependency resolution system for a trivial update.